DESCRIPTION

```diff
@@ -12,9 +12,10 @@ Encoding: UTF-8
 Imports:
   BAS,
   ggplot2,
+  ggrepel,
   jaspBase,
   jaspGraphs,
-  jfa
+  jfa (== 3.1.0)
```

Comment:
I'd recommend `<=` or `>=` (`>=` being much better, because you don't want to force older packages on people); having only a single version compatible is quite limiting

Reply:
Thanks. However, I need to rewrite a lot (more) for upgrading to jfa 0.4.0, and only 0.3.1 works with the module. Do I change this in that case? I can change this to `>= 0.4.0` when I do this rewriting.

Reply:
I guess if 3.1.0 is literally the only version that works with it, then your current implementation is the way to go